### DISCUSSION TITLE
How to parse DXF content?

### DISCUSSION THREAD

**Discussion by chenneng** [mozman/ezdxf#435] [Q&A]
I want know how to parsing the information in dxf file, is there any example?

**Comment by mozman** [comment]
Read the [documentation](https://ezdxf.mozman.at/docs/index.html) to learn how *ezdxf* works, start at the [beginners guide](https://ezdxf.mozman.at/docs/usage_for_beginners.html). There are also [tutorials](https://ezdxf.mozman.at/docs/tutorials/index.html) in the documentation and examples in the [repository](https://github.com/mozman/ezdxf/tree/master/examples).

**Reply by chenneng** [reply]
thank you
